scripts/examples/python/setup.py

```diff
@@ -0,0 +1,19 @@
+from setuptools import setup, find_packages
+
+setup(name='XenAPI',
+      version='1.2',
```

Comment:
It would make sense to keep the module version in sync with the API version defined in the datamodel_types.ml (current value is 2.14), even if the module does not change that often. I think it would be confusing for people if the module uses a version scheme different from the API (which is reflected in the API reference and the other SDK flavours). For now we can add it manually and later consider using a mechanism to autogenerated it (for example `ocaml/doc/jsapi.ml` and the SDK generator use mustache for wirting out the file).

Reply:
I may be able to extract the API version number from `setup.py` on build-time from `datamodel_types.ml`. I've hardcoded the version for the time being.